lib/urcli-money.js

```diff
@@ -66,8 +66,17 @@ function definePeriods () {
       }
       const yearAndMonth = arg < 9 ? `${year}-0${arg}` : `${year}-${arg}`
       let start = moment(yearAndMonth).utc()
-      let end = moment(yearAndMonth).utc().add(1, 'M')
-      periods.push([start, end])
+      // Check if month asked is the current month
+      let currentMonth = moment.utc().add(1, 'M').month()
```

Comment:
I think `let currentMonth = moment.utc().month() + 1` is more correct. Otherwise currentMonth refers to the next month, not the current one, which might cause confusion down the line. Beyond that you already have the current month in the variable `month` above. Is there any reason not to use it?